cmd/ctr/commands/content/fetch.go

```diff
@@ -185,13 +190,26 @@ func Fetch(ctx context.Context, client *containerd.Client, ref string, config *F
 		}
 	}
 
-	img, err := client.Fetch(pctx, ref, opts...)
+	var (
+		img     images.Image
+		imgPull containerd.Image
+		err     error
+	)
+
+	if config.UsePull {
+		imgPull, err = client.Pull(pctx, ref, opts...)
```

Comment:
`Pull` is same as `Fetch`, but with unpack handling. Should we just use `Pull` here without if check? This also  would let us to use `ctr` to "pull" from remote.